antarest/login/web.py

```diff
@@ -49,49 +49,81 @@ def auth() -> Any:
     def login() -> Any:
         return render_template("login.html")
 
-    @bp.route("/users", methods=["GET", "POST"], defaults={"id": None})
+    @bp.route("/users", methods=["GET"])
+    @jwt_required  # type: ignore
```

Comment:
Wrap the decorator inside another decorator to disable it when login is disabled in config. eg. https://stackoverflow.com/questions/5952641/decorating-decorators-try-to-get-my-head-around-understanding-it

Reply:
yes, i will do that in other issue, when we refactor storage